wfl/Profiles/RootProfile.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Hypermedia.Affordances;
+using Hypermedia.Transforms;
+using Hypermedia;
+using Symphono.Wfl.Controllers;
+using Symphono.Wfl.Models;
+
+namespace Symphono.Wfl.Profiles
```

Comment:
I agree this wouldn't be part of a Hypermedia namespace.  We could choose to have a FoodOrder namespace and Restaurant namespace etc. to have more fine grained folders.  I'm also fine if we delay that until we grow larger.  Personally I don't yet have an issue with what we have so far.